docs/core/diagnostics/cli-tools/dotnet-counters.md

```diff
@@ -0,0 +1,134 @@
+
+## Install dotnet-counters
+
+To install the latest release version of the `dotnet-counters` [NuGet package](https://www.nuget.org/packages/dotnet-counters):
+
+```
+dotnet tool install --global dotnet-counters
+```
+
+For details and other options, see [Installing the diagnostics tools](installing.md).
+
+## Using dotnet-counters
+
+*SYNOPSIS*
+
+```
+dotnet-counters [--version]
+                [-h, --help]
+                <command> [<args>]
+```
+
+*OPTIONS*
+
+```
+    --version
+        Display the version of the dotnet-counters utility.
+    -h, --help
+        Show command line help
+```
+
+*COMMANDS*
+
+```
+    list      Display a list of counter names and descriptions
+    monitor   Display periodically refreshing values of selected counters
+```
+
+*LIST*
+
+```
+    dotnet-counters list [-h|--help]
+
+    Display a list of counter names and descriptions, grouped by provider.
+
+    -h, --help
+        Show command line help
+
+    Examples:
+      > dotnet-counters list
+
+    Showing well-known counters only. Specific processes may support additional counters.
+    System.Runtime
+        cpu-usage                    Amount of time the process has utilized the CPU (ms)
+        working-set                  Amount of working set used by the process (MB)
+        gc-heap-size                 Total heap size reported by the GC (MB)
+        gen-0-gc-count               Number of Gen 0 GCs / sec
+        gen-1-gc-count               Number of Gen 1 GCs / sec
+        gen-2-gc-count               Number of Gen 2 GCs / sec
+        exception-count              Number of Exceptions / sec
+```
+
+*MONITOR*
+
+```
+    ### Examples:
+
+    1. Monitoring all counters from `System.Runtime` at a refresh interval of 3 seconds:
+
+      > dotnet-counters monitor --process-id 1902 System.Runtime
+
+    Press p to pause, r to resume, q to quit.
+      System.Runtime:
+        CPU Usage (%)                                 24
+        Working Set (MB)                            1982
+        GC Heap Size (MB)                            811
+        Gen 0 GC / second                             20
+        Gen 1 GC / second                              4
+        Gen 1 GC / Second                              1
+        Number of Exceptions / sec                     4
+
+    1. Monitoring just CPU usage and GC heap size from `System.Runtime` at a refresh interval of 5 seconds:
+
+      > dotnet-counters monitor --process-id 1902 System.Runtime[cpu-usage,gc-heap-size,exception-count]
```

Comment:
Remove exception-count or add it to the description of what the command does?

Reply:
Neither this command nor the command above explicitly set a refresh interval, but they seem to disagree about the what the default interval is?